src/tests/suite/daos_rebuild.c

```diff
@@ -120,6 +120,7 @@ rebuild_targets(test_arg_t **args, int args_cnt, d_rank_t *ranks,
 		if (args[i]->rebuild_cb)
 			args[i]->rebuild_cb(args[i]);
 
+	sleep(10); /* make sure the rebuild happens after kill */ 
```

Comment:
In cases that do not call this function with kill=true, does it make sense to insert the sleep? I'm not as familiar with the rebuild tests so it's not a strong objection, just asking since the comment mentions kill specifically.

Reply:
ah, sorry for confuse. the comment actually means kill & exclude, even only for excluding, it still needs to wait to make sure the rebuild start.